backendprocesses.h

```diff
@@ -4,6 +4,10 @@
 #undef unix
 #endif
 
+#ifdef unix
```

Comment:
Sorry to keep this PR going, but let's remove this. It's already done a few lines above